build.sbt

```
@@ -219,3 +219,16 @@ lazy val snapshot_generator = setupProject(
   localDependencies = Seq(internal_model, display, elasticsearch_typesafe),
   externalDependencies = CatalogueDependencies.snapshotGeneratorDependencies
 )
+
+/**
+ * To download form the wellcome maven repo in s3 locally, uncomment this
+ **/
+//import java.util.UUID
+//import com.amazonaws.auth.STSAssumeRoleSessionCredentialsProvider
+//s3CredentialsProvider := { _ =>
+//  val builder = new STSAssumeRoleSessionCredentialsProvider.Builder(
+//    "arn:aws:iam::760097843905:role/platform-read_only",
+//    UUID.randomUUID().toString
+//  )
+//  builder.build()
+//}
```

Would it work for you this block being formatted like this? Unfortunately my editor doesn't seem to like blocks of line comments in sbt files, so when I select the block and press my uncomment shortcut it doesn't work as expected.

```suggestion
// To download form the wellcome maven repo in s3 locally, uncomment this:
/* import java.util.UUID */
/* import com.amazonaws.auth.STSAssumeRoleSessionCredentialsProvider */
/* s3CredentialsProvider := { _ => */
/*   val builder = new STSAssumeRoleSessionCredentialsProvider.Builder( */
/*     "arn:aws:iam::760097843905:role/platform-read_only", */
/*     UUID.randomUUID().toString */
/*   ) */
/*   builder.build() */
/* } */
```

---

That breaks intelliJ